recipes/botan/all/conanfile.py

```diff
@@ -375,11 +447,17 @@ def _configure_cmd(self):
         if is_msvc(self):
             build_flags.append(f"--msvc-runtime={msvc_runtime_flag(self)}")
 
+        if self._is_glibc_older_than_2_25_on_linux and Version(self.version) >= '3.0':
+            # Botan 3.0+ requires glibc >= 2.25, but CCI's CI Linux images are
+            # currently older than that. Remove as soon as CCI updates!
```

Comment:
```suggestion
            # INFO: Botan 3.0+ requires glibc >= 2.25. Disable features to make it backward compatible
            # FIXME: CCI Docker images are running Ubuntu 16.04. Remove it after supporting later version.
```

Make it more explicit, so code editor will blink any FIXME in this repository.